INSTALL.md

```diff
@@ -0,0 +1,315 @@
+# Installation
+
+## Requirements
+* [Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or higher.
```

Comment:
I would drop the download url.  Anyone who got this far knows what Java is, and I would prefer to avoid implying that the choice of JDKs matters in general.

Reply:
I'd argue choice matters is there's something that wouldn't work on an earlier version. Is that the case?

Reply:
Sorry, here by "choice" of I meant "OracleJDK" vs "OpenJDK" vs "AzuJDKl" etc, not the version number

Reply:
Also, yes - it won't work on any version earlier than 1.8.